source/Ship.h

```diff
@@ -300,6 +300,8 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	double CloakingSpeed() const;
 	// If this ship should be immune to the next damage caused.
 	bool Phases(Projectile &projectile) const;
+	// Check if this ship has any jump capabilities.
+	bool HasDrive() const;
```

Comment:
Why here and not in ShipJumpNavigation? Do we need this at all if we can just access the navigation?

Reply:
I've renamed this to HasAnyDrive and moved it to the navigation class. 

> Do we need this at all if we can just access the navigation?

I think it's fine to have as shorthand for "has any jump method whatsoever" instead of needing to check both HasHyperdrive and HasJumpdrive.